### ISSUE TITLE
libsass not found

### ISSUE THREAD

**Issue by Kikobeats** [dlmanning/gulp-sass#215]
```bash
Error: `libsass` bindings not found. Try reinstalling `node-sass`?
```

Just in gulp-sass `1.3.3`. The problem is because is not possible to create the binary:

```bash
node scripts/build.js


module.js:340
    throw err;
          ^
Error: Cannot find module '/Users/josefranciscoverdugambin/Projects/kikobeats/kikobeats-blog/content/themes/Uno/node_modules/gulp-sass/node_modules/node-sass/node_modules/pangyp/bin/node-gyp'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:929:3
Build failed
```

**Comment by diagramatics** [comment]
Here too, running io.js and Windows 8.1. Seems like pangyp was introduced in node-sass 2.0.1, which is the version used for 1.3.3 (1.3.1 was using 2.0.0-beta). Problem is that it is introduced as a devDependency which won't be installed if you're installing node-sass as a dependency.

**Comment by Kikobeats** [comment]
not really

**Comment by benhansenslc** [comment]
Same issue here.

I was able to solve it temporarily by

    cd node_modules/gulp-sass/node_modules/node-sass
    npm install

**Comment by markau** [comment]
@diagramatics Windows 7 and io.js 1.3.0 here. Unfortunately no luck from @benhansenslc suggestion.

**Comment by Keats** [comment]
Try the 2.x branch

**Comment by tnguyen14** [comment]
Running into the same issue. @benhansenslc's suggestion works for node 0.12.1, but not for io.js 1.6.2.

**Comment by Keats** [comment]
@tnguyen14 can you try the 2.x branch ?

**Comment by tnguyen14** [comment]
I have `2.0.1`, is that different?
```
├─┬ grunt-sass@0.18.1
│ └── node-sass@2.0.1
└─┬ gulp-sass@1.3.3
  └── node-sass@2.0.1
```

**Comment by Keats** [comment]
Sorry, i meant gulp-sass 2.x branch (https://github.com/dlmanning/gulp-sass/tree/2.x)

**Comment by tnguyen14** [comment]
@Keats I just tried that, still give the same error for io.js.

**Comment by Keats** [comment]
Odd
Either way that's an error happening during the install of https://github.com/sass/node-sass so please report the bug on that repo

**Comment by jonjaques** [comment]
I found success on this issue by switching to the 2.x branch of this plugin—and then updating node-sass to 3.0.0-beta.4. The shipped binding for 3.0.0-alpha.1 is broken on Mac.

**Comment by Snugug** [comment]
I'm going to close this out then.
